### PR TITLE
Add PHPStan to the project

### PR DESCRIPTION
**Pull request by cmodijk**
TODO
- [x] Update `message-bus/tests`
- [x] Update `rabbitmq-bundle-bridge/tests`
- [x] Update `symfony-bridge/tests/tests`
- [x] Fix tests / bug's

### CODE REVIEW

File: packages/message-bus/src/Subscriber/NotifiesMessageSubscribersMiddleware.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function __construct(
             $this->logger = $logger;
         }
 
-        $this->level = $level;
+        $this->level = $level ?? LogLevel::DEBUG;
```
Comment (ruudk) [inline packages/message-bus/src/Subscriber/NotifiesMessageSubscribersMiddleware.php:32]: Same can be done for the logger:
```php
$this->logger = $logger ?? new NullLogger();
```

Comment (cmodijk) [reply packages/message-bus/src/Subscriber/NotifiesMessageSubscribersMiddleware.php:26]: Fixed

**Comment by ruudk** [comment]
@cmodijk PR is ready now. Could you please have a look?

**Comment by cmodijk** [comment]
@ruudk Looks good to me, i'm fine with merging and tagging this.

**Review by ruudk — APPROVED** [review]

**Pull request merged**